packages/host/app/components/operator-mode/create-file-modal.gts

```diff
@@ -531,11 +531,12 @@ export default class CreateFileModal extends Component<Signature> {
   private chooseType = restartableTask(async () => {
     this.clearSaveError();
     let isField = this.fileType.id === 'field-definition';
+
     this.selectedCatalogEntry = await chooseCard({
       filter: {
         on: catalogEntryRef,
         // REMEMBER ME
```

Comment:
i'm curious what we were supposed to remember here :-P

Reply:
Maybe @ef4 would know–it's an indicator introduced in https://github.com/cardstack/boxel/pull/1290 (Stop using async computed)